### PR TITLE
support embedding Capstone in Windows drivers

### PR DESCRIPTION
**Pull request by tandasat**
This update allows users to: 
- compile Capstone as a static driver library (capstone_static_winkernel.lib) using VS 2013 or later
- learn how to use Capstone in thier drivers with a sample project
- excersise all existing tests from Windows drivers with Capstone

We need following actions after this PR is merged.  
- addressing all compiler warnings on capstone_static_winkernel project
- removing myinttype.h
- dealing with uninvestigated below errors on MIPS Big-endian tests

> ERROR: Failed to disasm given code!
  
The first two were not done at this stage for ease of review, and the last one was not easy to address to me and did not look critical since there would not be so many situations to analyze MIPS code from drivers.

### CODE REVIEW

File: tests/test_winkernel.cpp [hunk 1]
```diff
@@ -0,0 +1,149 @@
+/* Second-Best Disassembler Engine */
+/* By Nguyen Anh Quynh <aquynh@gmail.com>, 2013-2016 */
```
Comment (aquynh) [inline tests/test_winkernel.cpp]: feel free to put your name (not my name) on any new files you created, thanks.

**Comment by aquynh** [comment]
i merged this to a new branch https://github.com/aquynh/capstone/tree/windriver. please base your future PR on this branch.

i think this can be merged soon after a bit more review, thanks.

File: tests/test_winkernel.cpp (hunk 1 above)
Comment (tandasat) [inline tests/test_winkernel.cpp]: Changed names in some files with 882052a. 

And understood about the new branch. Thanks for creating a new branch.

**Comment by aquynh** [comment]
picked & pushed your latest commit to the "windriver" branch, thanks!

**Comment by aquynh** [comment]
i cleaned the coding style at https://github.com/aquynh/capstone/commit/7518c3b23d5a512bffafdc194eddd63c181275b5, and removed myinttypes.h at https://github.com/aquynh/capstone/commit/b158b93a7d941a52bedd64ca6b78ff18de9b6ca3.

this looks good now. lets see if @brucedang has something to comment.

**Comment by tandasat** [comment]
Thanks for removing `myinttypes.h` and updating code. Those look good to me.

**Comment by zneak** [comment]
How's fuzzing going, by the way?

**Comment by tandasat** [comment]
@aquynh It has been a week since we asked @brucedang. I pinged him but have not gotten reply. Should we still wait for his comments?

**Comment by aquynh** [comment]
let me ping @brucedang one more time ...

**Comment by aquynh** [comment]
merged the "windriver" branch into "master", thanks!

now would you please port this over to the "next" branch?

**Comment by tandasat** [comment]
Thanks for merging it. I will 
- address warnings on master 
- confirm and file an issue ticket for errors in MIPS Big-endian tests
- then, port changes to the "next" branch

**Comment by aquynh** [comment]
that is a great plan, thanks!

**Pull request closed without merging**